_i18n/pt/home.html

```diff
@@ -0,0 +1,41 @@
+<div class="rich-text home-page">
```

Comment:
this entire file can be deleted (along with `_i18n/ru/home.html` etc.. because `index.md` renders all the translation keys directly. In some cases two different language files might have completely different structure, but here the structure is the same